src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Enums should be defined using capital letters. If you'd like to print print color names with small letters, you can provide an enum variable called colorName, which will be called in required place. 

```suggestion
public enum Colors {
  BLACK("black"),
  WHITE("white"),
  RED("red"),
  BLUE("blue");

  private final String colorName;

  // TODO remember to generate constructor and getter
}
```

Now in ColorSupplier class you can use created getter of the colorName:
```suggestion
  public String getRandomColor() {
    Colors[] colors = Colors.values();
    int index = random.nextInt(colors.length);
    return colors[index].getColorName();
  }
```